### ISSUE TITLE
Removed faces keep getting readded to cluster

### ISSUE THREAD

**Issue by illnesse** [nextcloud/recognize#391]
If i remove a face from a cluster and trigger recrawl, it gets readded to the same cluster, is that behaviour intentional?

**Comment by illnesse** [comment]
Actually, i am noticing faces keep getting readded over and over again without merging clusters. Is it possible there are multiple Recrawl Schedules running if you run occ recognize:recrawl multiple times?

**Comment by marcelklehr** [comment]
Yes, this is a known issue. As long as classifications are happening, clusters will be reset. I have a fix in mind already.

**Comment by illnesse** [comment]
I just noticed another issue after fixing a ton of face clusters, during second recrawl it seems some of my merged & named clusters are gone, they lost their name and are possibly split up into multiple clusters again. Is this related ?

**Comment by marcelklehr** [comment]
@illnesse This should be fixed as well with v3.1.0
